tests/__init__.py

```diff
@@ -20,7 +20,11 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
 
+import os
+
 from packit.utils import set_logging
 
 # debug logs from packit while testing
 set_logging(level=10)
+
+os.environ["RECORD_REQUESTS"] = "TRUE"
```

Comment:
so we can't change it?

Reply:
Do you think it's a problem? I would like to have it on by default.